### PR TITLE
DPL Analysis: make WritingCursor aware of how many rows were written

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

File: Framework/Core/include/Framework/AnalysisTask.h [hunk 1]
```diff
@@ -71,12 +71,20 @@ struct WritingCursor<soa::Table<PC...>> {
...
   {
     static_assert(sizeof...(PC) == sizeof...(T), "Argument number mismatch");
+    ++mCount;
     cursor(0, extract(args)...);
   }
 
+  /// Last index inserted in the table
+  size_t lastIndex()
+  {
+    return mCount;
```
Comment (qgp) [inline Framework/Core/include/Framework/AnalysisTask.h]: Looking at the index returned when going through collections,  just wondering whether this should not be:
```suggestion
    return mCount - 1;
```

Comment (ktf) [reply Framework/Core/include/Framework/AnalysisTask.h]: yes, it should.

Comment (ktf) [reply Framework/Core/include/Framework/AnalysisTask.h]: i will initialize to -1

**Comment by ktf** [comment]
@ginnocen I think you were asking for this as well..

**Pull request merged**